lib/open-search-types.d.ts

```diff
@@ -7,14 +7,34 @@ export interface DocumentItem {
   bitstreams: Bitstream[]
 }
 
-export interface SearchItem {
+export interface DocumentItemTerm {
```

Comment:
Given how a lot of these types are being used in `open-search-client`, you may want to consider defining them as zod schemas and using the inferred types from those schemas.